docs/source/interfaces/Indirect Settings.rst

```diff
@@ -0,0 +1,334 @@
+
+Options
+~~~~~~~
+
+Facility
+  Allows you to choose the selected facility.
+
+
+Indirect Bayes
+--------------
+
+This section is displayed only when the settings window is opened from the Bayes
+interface.
+
+Options
+~~~~~~~
+
+Restrict allowed input files by name
+  This will allow you to load input files with any names if unticked. Previously, only
+  input files with certain end extensions such as *_red* were allowed to be loaded. See
+  the glossary below to understand the restrictions on data when this option is ticked.
+
+Plot error bars for output plots
+  This will ensure that error bars are plotted on the output data plots if it is ticked.
+
+
+Data Analysis
+-------------
+
+This section is displayed only when the settings window is opened from the Data Analysis
+interface.
+
+Options
+~~~~~~~
+
+Restrict allowed input files by name
+  This will allow you to load input files with any names if unticked. Previously, only
+  input files with certain end extensions such as *_red* were allowed to be loaded. See
+  the glossary below to understand the restrictions on data when this option is ticked.
+
+Plot error bars for output plots
+  This will ensure that error bars are plotted on the output data plots if it is ticked.
+
+
+Data Corrections
+----------------
+
+This section is displayed only when the settings window is opened from the Data Corrections
+interface.
+
+Options
+~~~~~~~
+
+Restrict allowed input files by name
+  This will allow you to load input files with any names if unticked. Previously, only
+  input files with certain end extensions such as *_red* were allowed to be loaded. See
+  the glossary below to understand the restrictions on data when this option is ticked.
+
+Plot error bars for output plots
+  This will ensure that error bars are plotted on the output data plots if it is ticked.
+
+
+Data Reduction
+--------------
+
+This section is displayed only when the settings window is opened from the Data Reduction
+interface.
+
+Options
+~~~~~~~
+
+Restrict allowed input files by name
+  This will allow you to load input files with any names if unticked. Previously, only
+  input files with certain end extensions such as *_red* were allowed to be loaded. See
+  the glossary below to understand the restrictions on data when this option is ticked.
+
+Plot error bars for output plots
+  This will ensure that error bars are plotted on the output data plots if it is ticked.
+
+
+Other Interfaces
+----------------
+
+There are currently no interface-specific settings for the Diffraction, Simulation and Tools
+interfaces.
+
+
+Glossary of Allowed Suffixes
+----------------------------
+
+_red
+  This is produced in ISIS Energy Transfer on the :ref:`Data Reduction <interface-indirect-data-reduction>`
+  interface. It can also be produced from ISIS Calibration. It has an x axis of EnergyTransfer (meV) and a
+  y axis of Intensity (1/meV).
```

Comment:
Thanks for adding the axes. That Y axis is the intensity, I think is clear, but it would be nice to know also what is the vertical (i.e. the spectrum) axis of the 2D workspace; i.e. is it spectrum number, 2theta or Q?